file-formats/ints/type/zif_aff_ints_v1.intf.abap

```diff
@@ -0,0 +1,390 @@
+    "! Intelligent scenarios
+    BEGIN OF ty_main,
+      "! <p class="shorttext">Format Version</p>
+      "! Format version
+      "! $required
+      format_version   TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header           TYPE zif_aff_types_v1=>ty_header_60,
+      "! <p class="shorttext">Intelligent Scenario</p>
+      "! Intelligent Scenario
+      "! $required
+      islm_ps          TYPE ty_islm_ps,
+      "! <p class="shorttext">Objects based on Scenario</p>
+      "! Objects based on Scenario
+      islm_psddlobj    TYPE STANDARD TABLE OF ty_islm_psddlobj WITH DEFAULT KEY,
+      "! <p class="shorttext">Scenario Binding Mappings</p>
+      "! Scenario Binding Mappings
+      islm_psbindmp    TYPE STANDARD TABLE OF ty_islm_psbindmp WITH DEFAULT KEY,
+      "! <p class="shorttext">Dataset Bindings</p>
+      "! Dataset Bindings
+      islm_psbind      TYPE STANDARD TABLE OF ty_islm_psbind WITH DEFAULT KEY,
+      "! <p class="shorttext">Scenario Signatures</p>
+      "! Scenario Signatures
+      islm_pssig       TYPE STANDARD TABLE OF ty_islm_pssig WITH DEFAULT KEY,
+      "! <p class="shorttext">Signature Fields</p>
+      "! Signature Fields
+      islm_pssigfld    TYPE STANDARD TABLE OF ty_islm_pssigfld WITH DEFAULT KEY,
+      "! <p class="shorttext">Intelligent Scenario Modelling Context</p>
+      "! Intelligent Scenario Modelling Context
+      islm_mdlctxcc    TYPE STANDARD TABLE OF ty_islm_mdlctxcc WITH DEFAULT KEY,
+      "! <p class="shorttext">Modeling Context Filters</p>
+      "! Modeling Context Filters
+      islm_mctxfcc     TYPE STANDARD TABLE OF ty_islm_mctxfcc WITH DEFAULT KEY,
+      "! <p class="shorttext">Modeling Context Filters</p>
+      "! Modeling Context Filters
+      islm_pspardef    TYPE STANDARD TABLE OF ty_islm_pspardef WITH DEFAULT KEY,
+      "! <p class="shorttext">Scenario parameter signature</p>
+      "! Scenario parameter signature
+      islm_pspar       TYPE STANDARD TABLE OF ty_islm_pspar WITH DEFAULT KEY,
+      "! <p class="shorttext">Scenario additional information</p>
+      "! Scenario additional information
+      islm_psaddlinfo  TYPE STANDARD TABLE OF ty_islm_psaddlinfo WITH DEFAULT KEY,
+      "! <p class="shorttext">Scenario additional information texts</p>
+      "! Scenario additional information texts
+      islm_psaddlinfot TYPE STANDARD TABLE OF ty_islm_psaddlinfot WITH DEFAULT KEY,
+      "! <p class="shorttext">SBS Scenario Segment Fields</p>
+      "! SBS Scenario Segment Fields
+      islm_pssgmntfld  TYPE STANDARD TABLE OF ty_islm_pssgmntfld WITH DEFAULT KEY,
+      "! <p class="shorttext">Intelligent Scenario Models</p>
+      "! Intelligent Scenario Models
+      "! $required
+      islm_models      TYPE STANDARD TABLE OF ty_models WITH DEFAULT KEY,
+      "! <p class="shorttext">Model Prompt Templates</p>
+      "! Model Prompt Templates
```

Comment:
Descriptions must be sentence case: https://github.com/SAP/abap-file-formats/blob/main/docs/json.md#description

Reply:
Please check the rest of the description to see if they comply with the sentence case requirement